src/state/slices/assetsSlice/selectors.ts

```diff
@@ -72,7 +72,9 @@ const chainIdFeeAssetReferenceMap = (
             return ASSET_REFERENCE.Bitcoin
         }
       case CHAIN_NAMESPACE.Ethereum:
-        return ASSET_REFERENCE.Ethereum
+        return chainReference === CHAIN_REFERENCE.AvalancheCChain
```

Comment:
i have been updating these to a switch with exhaustive cases and a default throw for an invalid chain reference to be more explicit than default fallback

Reply:
Yeah that is more extensible. for sure, updated in https://github.com/shapeshift/web/pull/2164/commits/17072ffbb088ec0687b9969411df34e7b08c98c9